tests/ui/while_let_on_iterator.rs

```diff
@@ -259,7 +260,7 @@ fn issue1924() {
         fn f(&mut self) -> Option<u32> {
             // Used as a field.
             while let Some(i) = self.0.next() {
-                if i < 3 || i > 7 {
```

Comment:
Why are we changing this one?

Reply:
It was triggering `manual_range_contains`. Since the change doesn't affect what the test is checking for I just applied the suggestion.